authorization.md

```diff
@@ -198,3 +198,9 @@ Authorization: Bearer <access token>
 
 #### Response
 This endpoint will respond with a `204 No Content` HTTP code.
+
+## Example code
+
+Repository: https://github.com/iZettle/example-integration
+
+We have produced a pair of front and back end apps to show how to get started with the Authorization service as an external developer. It initiates an Authorization Code Grant flow, retrieves a token and provides it via a session cookie to the web app to send with requests to other Zettle APIs. There is also code in the server app to refresh tokens when they appear to have expired.
```

Comment:
The example integration consists of a web app (frontend) and a server app (backend). It initiates an Authorization Code Grant flow where an access token is retrieved and provided through a session cookie to the web app to send requests to other Zettle APIs. When an access token expires, the server app will use the refresh token to generate a new access token.

To get started with the Authorization Code Grant flow, [try the example integration](https://github.com/iZettle/example-integration).